_overviews/core/nightlies.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: singlepage-overview
+title: Nightly Versions of Scala
+permalink: /overviews/core/:title.html
+---
+
+We regularly publish nightly versions of both Scala 2 and 3 so that users can preview and test the contents of upcoming releases.
+
+We informally call them "nightly" versions, but technically it's a misnomer. A so-called “nightly” is built for every merged PR.
+
+Here's how to find and use these versions.
+
+## Scala 3
+
+Scala 3 nightly versions are published to Maven Central. If you know the full version number of the nightly you want to use, you can use it just like any other Scala 3 version.
+
+One quick way to get that version number is to visit https://dotty.epfl.ch and look in the upper left corner.
+
+Another way is to scrape Maven Central, as shown in this script: https://raw.githubusercontent.com/VirtusLab/community-build3/master/scripts/lastVersionNightly.sc
+
+A third way is to use [scala-cli](https://scala-cli.virtuslab.org), as follows.
+
+### scala-cli
+
+You can run nightlies with commands such as:
+
+    scala-cli -S 3.nightly
+    scala-cli -S 3.3.nightly
+
+The default command is `repl`, but all the other scala-cli subcommands such as `compile` and `run` work, too. It also works with `//>` directives in your script itself, for example:
+
+    //> using scala 3.nightly
+
+See this [scala-cli doc page](https://scala-cli.virtuslab.org/docs/commands/compile#scala-nightlies) for details.
+
+## Scala 2.13 or 2.12
+
+Scala 2 nightly versions are published to a special resolver. Unless you are using scala-cli, you'll need to add that resolver to your build configuration in order to use these versions.
+
+### quick version (sbt)
+
+    Global / resolvers += "scala-integration" at
+      "https://scala-ci.typesafe.com/artifactory/scala-integration/"
+    scalaVersion := "2.13.15-bin-abcd123"
+
+For a 2.12 nightly, substitute e.g. `2.12.20` for `2.13.15`; in either case, it's the version number of the _next_ release on that branch.
+
+For `abcd123`, substitute the first 7 characters of the SHA of the latest commit to the [2.13.x branch](https://github.com/scala/scala/commits/2.13.x) or [2.12.x branch](https://github.com/scala/scala/commits/2.12.x) that has a green checkmark. (Clicking the checkmark will show a CI job name with the whole version in its name.)
+
+A quick way to find out the full version number of a current nightly is to use [scala-cli](https://scala-cli.virtuslab.org), as follows.
+
+### quick version (scala-cli)
+
+You can run nightlies with:
+
+    scala-cli -S 2.13.nightly
+    scala-cli -S 2.nightly     # same as 2.13.nightly
+    scala-cli -S 2.12.nightly
+
+The default command is `repl`, but all the other scala-cli subcommands such as `compile` and `run` work, too. It also works with `//>` directives in your script itself, for example:
+
+    //> using scala 2.nightly
```

Comment:
I did it that on purpose because I figure either someone is interested in 3, or they're interested in 2, so each section ought to be complete in itself.

Whether it's really best that way, I don't have a strong feeling about. Sounds like your feeling isn't that strong either, so let's leave it like it is for now. I wouldn't mind if it got changed in the future.